news/2023/2023-05-01-the-osu-legacy-of-reesewashere.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "The osu! YouTube legacy of ReeseWasHere"
+date: 2023-05-01 10:00:00 +0000
+---
+
+Before livestreaming and player YouTube channels were widespread, you could rely only on one man with a camera...
+
+![](/wiki/shared/news/2023-05-01-osu-legacy-of-reesewashere/banner.jpg)
+
+If you were even slightly engaged with the osu! community between 2015 and 2016, you probably heard of [ReeseWasHere](https://www.youtube.com/@ReeseWasHere), seen some of his YouTube videos, or at the very least seen the saying "Reese get the camera!!" whenever a top player would set a nice score. If you ever saw the ["mom get the camera"](https://www.youtube.com/watch?v=mLRbZJS5A_E) meme from the Call of Duty community, Reese would be the osu! equivalent to that. So without further ado, let's get into the history of this legendary community member and see what they are up to now!
```

Comment:
```suggestion
If you were even *slightly* engaged with the osu! community between 2015 and 2016, you might've heard of [ReeseWasHere](https://www.youtube.com/@ReeseWasHere). You may have even seen some of his YouTube videos, or at the very least caught wind the saying *"Reese get the camera!!"* at the mention of an amazing high score (in homage to the [similar meme from the Call of Duty scene](https://www.youtube.com/watch?v=mLRbZJS5A_E)). So without further ado, let's get into the history of this legendary community member and see what they are up to now!
```

Reply:
```suggestion
If you were even *slightly* engaged with the osu! community between 2015 and 2016, you might've heard of [ReeseWasHere](https://www.youtube.com/@ReeseWasHere). You may have even seen some of his YouTube videos, or at the very least caught wind of the saying *"Reese get the camera!!"* at the mention of an amazing high score (in homage to the [similar meme from the Call of Duty scene](https://www.youtube.com/watch?v=mLRbZJS5A_E)). So without further ado, let's get into the history of this legendary community member and see what they are up to now!
```

Reply:
the of is there? not really following

Reply:
What you suggested:
![image](https://github.com/ppy/osu-wiki/assets/24534022/b613d611-adea-4ba4-9926-38edf9e7f5df)
I edited it to:
![image](https://github.com/ppy/osu-wiki/assets/24534022/e542f80e-c902-469b-904d-001e3cc61721)

Just wanted to check if you agree with that?

Reply:
'caught wind of' is correct, though I'm fairly sure that's what I suggested too